### PR TITLE
Fastlane to take screenshots in various langs and simulators.

### PR DESCRIPTION
**Pull request by montehurd**
https://phabricator.wikimedia.org/T194790

Uses [fastlane snapshot](https://docs.fastlane.tools/actions/snapshot/) with UITesting to capture screenshots of the app on various devices and languages.

Devices:
- iPhone 5s
- iPhone 6
- iPhone 6 Plus
- iPhone X
- iPad Air
- iPad Pro (12.9-inch)

Languages:
- en-US
- de-DE
- es-ES
- fr-FR
- it 
- ja
- nl-NL
- pt-PT
- ru 
- sv
- zh-Hans
- da
- el
- fi
- id
- ko
- ms
- th
- tr
- vi

What this isn't:
This is not visual diff or visual test related functionality.

What this is:
Think of this as "printing the app", for easier proofing of release candidates.

Questions:
- can we kick this off once a day? (probably at night since it takes a while to generate all screenshots)
- can we open a port on the `buildmini` so we can access the screenshot index file externally?

Here's a gif of the index html file fastlane snapshot generates. Note that this is only for one device (iPhone 5s), whereas the PR captures screenshots for 6 devices.

480 mov

[Higher res](https://i.imgur.com/AowFwBI.mp4).

Now that I have a handle on this I think I can probably update it to capture almost every screen in the app :)

### CODE REVIEW

File: Wikipedia/Code/WMFExploreViewController.m [hunk 1]
```diff
@@ -404,7 +404,8 @@ - (void)updateFeedSourcesUserInitiated:(BOOL)wasUserInitiated completion:(nonnul
     }
     self.loadingNewContent = YES;
     if (!self.refreshControl.isRefreshing) {
-        [self.refreshControl beginRefreshing];
+        // TODO: fix this - causing UITest to hang for a few minutes. See https://stackoverflow.com/questions/40983184/xcode-8-ui-testing-taking-very-long
```
Comment (montehurd) [inline Wikipedia/Code/WMFExploreViewController.m]: This needs to be fixed before merging.

**Comment by joewalsh** [comment]
retest this please

**Comment by montehurd** [comment]
retest this please

File: Wikipedia.xcodeproj/xcshareddata/xcschemes/Wikipedia.xcscheme [hunk 2]
```diff
@@ -86,6 +87,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      disableMainThreadChecker = "YES"
```
Comment (joewalsh) [inline Wikipedia.xcodeproj/xcshareddata/xcschemes/Wikipedia.xcscheme]: Can this be removed and only have the change on the "Test" config?

Comment (montehurd) [reply Wikipedia.xcodeproj/xcshareddata/xcschemes/Wikipedia.xcscheme]: I'll try...

File: Wikipedia/Code/WMFExploreViewController.m (hunk 1 above)
Comment (montehurd) [reply Wikipedia/Code/WMFExploreViewController.m]: Fixed! See https://github.com/wikimedia/wikipedia-ios/pull/2314/commits/7ec6fef6288e416d58ffde8cb5f76138789df9da

File: Wikipedia.xcodeproj/xcshareddata/xcschemes/Wikipedia.xcscheme (hunk 2 above)
Comment (montehurd) [reply Wikipedia.xcodeproj/xcshareddata/xcschemes/Wikipedia.xcscheme]: Fixed here: https://github.com/wikimedia/wikipedia-ios/pull/2314/commits/1238c8cd7e13e6f4a12d71c94f17e79dcf8ede59

**Pull request merged**